docs/context/ref/ChatMessage.md

```diff
@@ -0,0 +1,51 @@
+---
+id: ChatMessage
+sidebar_label: ChatMessage
+title: ChatMessage
+hide_title: true
+---
+# `ChatMessage`
+
+Message to send to a chat.
+
+## Type
+
+`fdc3.chat.message`
+
+## Schema
+
+- https://fdc3.finos.org/schemas/next/chatMessage.schema.json
+
+## Details
+
+| Property    | Type    | Required | Example Value     |
+|-------------|---------|----------|-------------------|
+| `type`      | string  | Yes      | `'fdc3.chat.message'`  |
+| `chatRoom`      | ChatRoom  | Yes       | `{ type: 'fdc3.chat.room', providerName: 'Symphony', id:{ roomdId: 'j75xqXy25NBOdacUI3FNBH'} }`      |
+| `message`  | [Message](https://fdc3.finos.org/schemas/next/message.schema.json)  | Yes       | `'A message to send'` |
```

Comment:
The `Message` type is added in another PR https://github.com/finos/FDC3/pull/779
(See [schema](https://github.com/finos/FDC3/blob/75141ff3561c22399b4b7dd34795d35f5f1307c5/src/context/schemas/message.schema.json) )